aws_lambda_powertools/utilities/idempotency/config.py

```diff
@@ -0,0 +1,43 @@
+from typing import Dict
+
+
+class IdempotencyConfig:
+    def __init__(
```

Comment:
Looks great - Hopefully when 3.6 is EOL we'll be able to move to dataclasses to make this easier too, including having a generic Config that auto-discovers env vars based on config option name

Reply:
@heitorlessa should we add a mini project for ideas?